lnwallet/channel.go

```diff
@@ -3504,44 +3545,12 @@ func (lc *LightningChannel) computeView(view *htlcView, remoteChain bool,
 	// We evaluate the view at this stage, meaning settled and failed HTLCs
 	// will remove their corresponding added HTLCs.  The resulting filtered
 	// view will only have Add entries left, making it easy to compare the
-	// channel constraints to the final commitment state.
+	// channel constraints to the final commitment state. If any fee
```

Comment:
As the fee rate is included in the returned view now, we no longer need to return the fee rate as a distinct parameter.

Reply:
It is not returned as distinct parameter. I just set it to its own variable to minimize the diff here.

Reply:
Ah, I see what you mean now. `feePerKw` is returned from `computeView`. Fixed!